lib/adapter.js

```diff
@@ -1117,8 +1106,9 @@ module.exports = (function() {
    * @param  {Function} fn
    * @param  {[type]}   cb
    */
-  function spawnConnection(connectionName, fn, cb) {
+  function spawnConnection(queryType, connectionName, fn, cb) {
```

Comment:
Looked like an internal call, but someone else may be more aware of the ramifications.